nginx-nodejs-redis/docker-compose.yml

```
@@ -0,0 +1,25 @@
+
+services:
+  redis:
+    image: 'redislabs/redismod'
+    ports:
+      - '6379:6379'
+  web1:
+    restart: on-failure
+    build: ./web1
+    ports:
+      - '81:5000'
+  web2:
+    restart: on-failure
+    build: ./web2
```

Same here, use the `web` directory to build the image and setup a dedicated `hostname`
```suggestion
  web2:
    restart: on-failure
    build: ./web
    hostname: web2
```